proto/api/v1/metadata_service.proto

```diff
@@ -93,4 +117,10 @@ service MetadataService {
             get: "/v1/backup/status"
         };
     }
+
+    rpc GetCentralCapabilities(Empty) returns (CentralServicesCapabilities) {
+        option (google.api.http) = {
+            get: "/v1/central-capabilities"
```

Comment:
I would said generally we try and avoid dashes in urls; though not a hard and fast rule

Reply:
I disagree about dashes in URLs.

URLs where the endpoint name is "Multiple Words" should be all lowercase and the spaces should be replaced with dashes.

This practice grew out of Google's recommendations for SEO, but the principles it is based on apply to API URLs as well.

1. It's easier to understand on first reading, especially for ESL folks. (compare "get-network-baselines" vs. "getnetworkbaselines")
2. It easier to remember, because you can "tokenize" it in your head.

A lot of our older endpoints do `runallthewordstogether`, but we should `do-it-a-better-way` from now on. ;)